spec/controllers/stats_controller_spec.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe StatsController, type: :controller do
+  let(:user) { FactoryBot.create(:user) }
+
+  let(:video) { FactoryBot.create(:video) }
+
+  let(:video_watch_event) { FactoryBot.create(:video_watch_event, video_id: video.id, user_id: user.id) }
+  let(:video_watch_event2) { FactoryBot.create(:video_watch_event, video_id: video.id, user_id: user.id, duration: 2) }
```

Comment:
We usually use 1snake_case1 for numbers as well:

```suggestion
  let(:video_watch_event_2) { FactoryBot.create(:video_watch_event, video_id: video.id, user_id: user.id, duration: 2) }
```